modules/codegen-plugin/src/sbt-test/codegen-plugin/aws-newtype-flatten/src/main/scala/Main.scala

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
If we decide that this test is good enough, I'll add a similar one to the mill plugin

Reply:
@denisrosca because we can share the logic between SBT/mill, I'm happy with just having this test, no need for a mill-specific test.